### PR TITLE
[LDAP Wizard] make all the buttons beautiful

### LINKED ISSUE #18956: "Clear Username-LDAP User Mapping" & "Clear Groupname-LDAP Group Mapping" should be coloured border buttons
### Steps to reproduce

1. Enable LDAP app.
2. Configure a LDAP and click on expert tab

### Expected behaviour
"Clear Username-LDAP User Mapping" & "Clear Groupname-LDAP Group Mapping" should be border coloured buttons due to they appear among a lot of text

### Actual behaviour
"Clear Username-LDAP User Mapping" & "Clear Groupname-LDAP Group Mapping"  seem to be part of the text instead two buttons.

### Server configuration
**Operating system**:
Ubuntu 14.04

**Web server:**
Apache

**Database:**
MySQL

**PHP version:**
5.5.9

**ownCloud version:** (see ownCloud admin page)
master version:"8.2.0.4","versionstring":"8.2 pre alpha"

**Updated from an older ownCloud or fresh install:**
Fresh

**The content of config/config.php:**

```

```

**Are you using external storage, if yes which one:** local/smb/sftp/...
No

**Are you using encryption:** 
No

**Logs**
```
```
### Client configuration

**browser**
Firefox
botonesldap

### PR DESCRIPTION
**Pull request by blizzz**
Fixes #18956 

Not only the Buttons on the Expert tab were affected, but also on the other tabs.

Please test and review @davitol @jancborchardt @PVince81 

Also, should this go back to 8.2? Currently that really looks ugly. In 8.1 they don't look great and different as well, but at least they are clearly visible as buttons.

### CODE REVIEW

**Comment by davitol** [comment]
:-1: Using Firefox as browser, the design is broken

screen shot 2015-11-05 at 14 05 22

File: apps/user_ldap/js/wizard/view.js [hunk 1]
```diff
@@ -349,8 +349,7 @@ OCA = OCA || {};
 		render: function () {
 			$('#ldapAdvancedAccordion').accordion({ heightStyle: 'content', animate: 'easeInOutCirc'});
 			this.$settings.tabs({});
-			$('.ldap_submit').button();
-			$('.ldap_action_test_connection').button();
+			$('#ldapSettings button').not('.icon-default-style').button();
```
Comment (PVince81) [inline apps/user_ldap/js/wizard/view.js]: protip, you can also use `#ldapSettings button:not(.icon-default-style)` :smile:

**Comment by blizzz** [comment]
@davitol good catch, please retry.

**Comment by davitol** [comment]
:+1: WFM Just only two comments to let you know:
1) The arrows are now wider than before the PR
screen shot 2015-11-05 at 17 02 58
2) In every LDAP wizard tab, if you mouse over the "descriptions" of each field, the cursor turns into a finger cursor, that makes nonsense. This behaviour was the same before the fix.
screen shot 2015-11-05 at 17 13 50


Feel free to decide if it's worth fixing these low-severity "bugs"

**Comment by blizzz** [comment]
> The arrows are now wider than before the PR

Too many sweets, they should go on a diet. Especially since the winter days are only coming. Or do more sports, that's usually the better way.

> In every LDAP wizard tab, if you mouse over the "descriptions" of each field, the cursor turns into a finger cursor, that makes nonsense. This behaviour was the same before the fix.

Probably because their are labels, text input fields would get the focus usually, but that does not work with the multiselect fields. I am confident we can override this behaviour, but out of this scope. You may want to open a new issue for this ;)

**Comment by MorrisJobke** [comment]
Tested and works :+1:

**Comment by blizzz** [comment]
@karlitschek do we want to backport (8.2 only)?

**Comment by karlitschek** [comment]
please backport

**Comment by blizzz** [comment]
Backport to 8.2: #20410

**Pull request merged**